src/components/ExplainSideColumn.tsx

```diff
@@ -40,7 +40,9 @@ export const ExplainSideColumn = (props:any) => {
             <img alt="back" src={backArrow}/>
           </span></NavLink>
         </div>
-        <img src={imgUrl} alt={""}></img>
+        <div style={{display:"flex", justifyContent: "center"}}>
+          <img src={imgUrl} alt={""}/>
```

Comment:
Should provide a text for the alt attribute.

Reply:
done